### PR TITLE
GitActions for unit test

### PR DESCRIPTION
**Pull request by LohithNarayan**
Please choose Workflow name> GitActions for unit test

[python-app.yml](https://github.com/LohithNarayan/ringcentral-python/blob/master/.github/workflows/python-app.yml)

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@ coveralls>=1.1
 discover>=0.4.0
 requests-mock>=1.2.0
 setuptools
-python-dotenv>=1.0.0
+python-dotenv>=0.1.0
```
Comment (tylerlong) [inline requirements-dev.txt:6]: What is the reason for this change?

Comment (LohithNarayan) [reply requirements-dev.txt:6]: I got an error stating there is no version 1.0.0 found, suggestions were0.1.0 till 0.x.x

File: .github/workflows/python-app.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
```
Comment (tylerlong) [inline .github/workflows/python-app.yml:21]: Remove 3.7, since it will not be supported officially.

Comment (LohithNarayan) [reply .github/workflows/python-app.yml:21]: ok sure

**Review by LohithNarayan — COMMENTED** [review]
change to Dotenv - demo_subscription.py

File: demo_mms.py [hunk 3]
```diff
@@ -31,7 +41,7 @@ def main():
     builder = sdk.create_multipart_builder()
     builder.set_body({
         'from': {'phoneNumber': mms_number},
-        'to': [{'phoneNumber': MOBILE}],
+        'to': [{'phoneNumber': phone_number}],
```
Comment (tylerlong) [inline demo_mms.py:44]: Please change to `env['RINGCENTRAL_RECEIVER']`

File: demo_mms.py [hunk 4]
```diff
@@ -1,12 +1,22 @@
...
 def main():
-    sdk = SDK(APP_KEY, APP_SECRET, SERVER)
+    #sdk = SDK(APP_KEY, APP_SECRET, SERVER)
+    sdk = SDK(env['RINGCENTRAL_CLIENT_ID'], env['RINGCENTRAL_CLIENT_SECRET'], env['RINGCENTRAL_SERVER_URL'])
     platform = sdk.platform()
-    platform.login(USERNAME, EXTENSION, PASSWORD)
+    platform.login(jwt = env['RINGCENTRAL_JWT_TOKEN'])
+
+    params = {'from': {'phoneNumber': env['RINGCENTRAL_USERNAME']},'to': [{'phoneNumber': env['RINGCENTRAL_RECEIVER']}],'text': "MMS message"}
+    response = platform.post('/restapi/v1.0/account/~/extension/~/mms', params)
```
Comment (tylerlong) [inline demo_mms.py:16]: This line is to send the mms? Please double check the logic.

**Review by tylerlong — CHANGES_REQUESTED** [review]
- Please try `python-dotenv>=1.0.0` again, if it doesn't work, add comments to describe the details.
- Please manually run each demo_*.py files to verify that they work
- double check your change to demo_mms.py, I think the code send the message twice.
